src/components/Button/styles.module.css

```diff
@@ -1,6 +1,18 @@
 .button {
+  /*fall back presets */
+  --selectedColor: var(--safe-grey);
+
+  --lighten-100: var(--button-lighten-safe-grey-100);
+  --lighten-200: var(--button-lighten-safe-grey-200);
+  --darken-100: var(--button-darken-safe-grey-100);
+  --darken-200: var(--button-darken-safe-grey-200);
```

Comment:
I think my biggest remaining question is if `-100` and `-200` are the best naming scheme, since they don't really say *what* the colors are for. In fact, I don't fully grasp what they're for. Looks like active and hover states? Would `-hover` and `-active` work?

Reply:
mhm, yeah good suggestion. i've changed the variable names to -hover and -active, for a better association and clarity. I forgotten, but now I also removed the 'darken' variables, since the last request for better uniformity.